### PR TITLE
feature: health check supports custom host port.

### LINKED ISSUE #1811: request help: Health Checks can support customized PORT ?
sometimes http work and health check  listen on different ports  ,

like spring boot cloud 
http api request works on 8080,
but actuator health works on 8888 port

so  a router config like :

```"checks": {
            "active": {
                "http_path": "/actuator/health",
                "port": "8888",
                "host": "test",
                "healthy": {
                    "interval": 2,
                    "successes": 1
                },
                "unhealthy": {
                    "interval": 1,
                    "http_failures": 2
                }
            },
            "passive": {
                "healthy": {
                    "http_statuses": [200],
                    "successes": 3
                },
                "unhealthy": {
                    "http_statuses": [503,404],
                    "http_failures": 3,
                    "tcp_failures": 3
                }
            }
```

i did not find any health port define in https://github.com/apache/incubator-apisix/blob/master/apisix/schema_def.lua#L81

is it  supported now ?

_Originally posted by @souzens in https://github.com/apache/incubator-apisix/issues/1809#issuecomment-654671064_

### PR DESCRIPTION
**Pull request by shuaijinchao**
### What this PR does / why we need it:
FIX #1811 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Comment by membphis** [comment]
add test cases about invalid `port`, eg: `port = -1` or `port = 9999999`

File: apisix/schema_def.lua [hunk 1]
```diff
@@ -78,6 +78,11 @@ local health_checker = {
                 timeout = {type = "number", default = 1},
                 concurrency = {type = "integer", default = 10},
                 host = host_def,
+                port = {
+                    type = "integer",
+                    minimum = 1000,
```
Comment (membphis) [inline apisix/schema_def.lua:83]: I think the minimum value should be `1`.

Comment (shuaijinchao) [reply apisix/schema_def.lua:83]: I think that in general, ports below 1000 are open to the system, so here is set to 1000.

**Comment by shuaijinchao** [comment]
> add test cases about invalid `port`, eg: `port = -1` or `port = 9999999`

OK, no problem

File: apisix/schema_def.lua (hunk 1 above)
Comment (membphis) [reply apisix/schema_def.lua:83]: for example the HTTP port `80` is fine.

Comment (shuaijinchao) [reply apisix/schema_def.lua:83]: OK, I will update.

**Comment by shuaijinchao** [comment]
@membphis already fixed, please recheck.

**Comment by membphis** [comment]
@shuaijinchao merged, many thx

**Pull request merged**